tests/e2e/config/jest.config.js

```diff
@@ -32,4 +32,5 @@ module.exports = {
 	],
 	testPathIgnorePatterns: [ '<rootDir>/tests/e2e/specs/performance' ],
 	transformIgnorePatterns: [ 'node_modules/(?!(woocommerce)/)' ],
+	testMatch: [ '**/?(*.)+(spec|test).[jt]s?(x)' ],
```

Comment:
Currently, we're using the config from `@wordpress/scripts`, which is:

```
	testMatch: [ '**/specs/**/*.[jt]s?(x)', '**/?(*.)spec.[jt]s?(x)' ],
```

I changed it to use one of Jest's [default match expressions](https://jestjs.io/docs/configuration#testmatch-arraystring).